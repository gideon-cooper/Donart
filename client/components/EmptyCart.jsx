import React from 'react'
import { Link } from 'react-router-dom'

export default function EmptyCart () {
  return (
    <div className="emptyCart">
      <h1>Cart is empty :'( Try adding something!</h1>
      <br></br>
      <Link to="/">Home</Link>
    </div>
  )
}
