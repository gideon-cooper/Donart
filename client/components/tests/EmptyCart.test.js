import React from 'react'
import { HashRouter as Router } from 'react-router-dom'
import renderer from 'react-test-renderer'

import EmptyCart from '../EmptyCart'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <EmptyCart />
      </Router>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})