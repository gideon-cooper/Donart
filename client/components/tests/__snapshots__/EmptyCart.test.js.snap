// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="emptyCart"
>
  <h1>
    Cart is empty :'( Try adding something!
  </h1>
  <br />
  <a
    href="#/"
    onClick={[Function]}
  >
    Home
  </a>
</div>
`;
